packages/sourcecred/src/plugins/discord/reactionWeights.js

```diff
@@ -16,16 +16,16 @@ export type ChannelWeightConfig = {|
 |};
 
 export type EmojiWeightMap = {[Model.Snowflake]: NodeWeight};
-export type EmojiWeightConfig = {|
-  +defaultWeight: NodeWeight,
+export type ReactionWeightConfig = {|
   +weights: EmojiWeightMap,
+  +defaultWeight: NodeWeight,
   +applyAveraging: boolean,
 |};
 
 export type WeightConfig = {|
   +roleWeights: RoleWeightConfig,
   +channelWeights: ChannelWeightConfig,
-  +emojiWeights: EmojiWeightConfig,
+  +emojiWeights: ReactionWeightConfig,
```

Comment:
I would say for consistency's sake we might change this to `reactionWeights`, if we're making the word emoji obsolete, although I could see an argument where this might be over complicating the migration

Reply:
Yeah I chose not to for now because that attribute has deep and broad tendrils that would be more work to refactor. Some other time.

Reply:
Sounds great!